api/graphql/mutations.js

```diff
@@ -0,0 +1,14 @@
+import {
+  convertGraphqlCreateData,
+  createPost as underlyingCreatePost,
+  validateGraphqlCreateData
+} from '../models/post/util'
+
+export const updateMe = (userId, changes) =>
+  User.find(userId)
+    .then(user => user.validateAndSave(changes))
+
+export const createPost = (userId, data) =>
+  validateGraphqlCreateData(data)
```

Comment:
maybe convert and then validate? that way, the conversion function can live here (or at least somewhere within the graphql folder) because it's graphql-specific, and then validation can become independent of the input context and reusable for Post instances in general.